securing_apps/topics/oidc/java/params_forwarding.adoc

```diff
@@ -23,6 +23,8 @@ enter the scope options `address phone`, then the request to {project_name} will
 
 * prompt - {project_name} supports these settings:
 ** `login` - SSO will be ignored and the {project_name} login page will be always shown, even if the user is already authenticated
+** `consent` - Applicable only for the clients with `Consent Required`. If it is used, the Consent page will be always displayed
+even if user already granted consent before to this client.
```

Comment:
I suggest the following for the second sentence:
If it is used, the Consent p[age will always be displayed, even if the user previously granted consent to this client.